CHANGELOG.md

```diff
@@ -9,9 +9,34 @@ This changelog attempts to conform to the changelog spec on [keepachangelog.org]
 The evergreen, canonical changelog for *all NILRT branches* can be [found here](https://github.com/ni/nilrt/blob/HEAD/CHANGELOG.md).
 
 ## Latest Updates
-* nilrt: 301
-* meta-nilrt: 812
-* linux: 215
+* nilrt: 307
+* meta-nilrt: 852
+* linux: 235
+
+
+----
+## 11.2
+Branch: `nilrt/25.5/scarthgap`
+
+### nilrt
+No changes.
+
+### meta-nilrt
+
+
+#### Added
+- [Added](https://github.com/ni/meta-nilrt/pull/835) support for wired `802.1X`.
+- [Added](https://github.com/ni/meta-nilrt/pull/839) `nilrt-logging` to install INI file to specify log paths.
+- [Added](https://github.com/ni/meta-nilrt/pull/840) persistent logging.
+- [Added](https://github.com/ni/meta-nilrt/pull/842) nilrt container images.
+- [Added](https://github.com/ni/meta-nilrt/pull/852) scripts for limited tracefs access.
+
+#### Deprecated
+- [Removed](https://github.com/ni/meta-nilrt/pull/838) System.map file from boot dir.
+
+#### Fixed
+- [Fixed](https://github.com/ni/meta-nilrt/pull/826) error output when provisioning mmc targets.
+- [Fixed](https://github.com/ni/meta-nilrt/pull/833) coreutils package priority value.
```

Comment:
I'd have called this minor change that's not useful to customers but apparently a customer brought up the issue so I guess ok to add it here.